### ISSUE TITLE
Mermaid diagram for raster-clip isn't working on GitHub markdown

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/alpenglow#3]
If you browse https://github.com/phetsims/alpenglow/blob/main/doc/raster-clip.md, it just shows a not-very-helpful error.

@marlitas, any thoughts on what I'm doing wrong here? It is working in the WebStorm/Intellij preview, AND the [Mermaid live editor](https://mermaid.live/edit#pako:eNq1Wd1v2zYQ_1cEFQ1swMFMZXsRhr4kKRCgBdp4G9bZfmAsuhYqU4JEoWmD_O_jhyQeeZQ1N10eAvGO97tP3VH0U7wrMxan8b4ov-4OtBbRHzcbHsk_839X0Ka5YfsoY3vaFiLal1xcNvl3lv66rB6DW8tWVK24VstonxdFGr3aU7qIGlGXX1j6ii6XiyDOgJBzH4DuAcCSTgEc8owBeVFT3lS0ZlwMKIAGvWjah881rQ7GhiZab-JoE28NU_0Z2w4t_9JIHlj9_lC_uaeNYLVer7dSLE1T64uHcZt9ZgOEXlgEtRwDYDwD3vZ2Dh5vuOECy6LLyzfRHc9FTgtNma3hajs3EpCmRXZFXlUsMyDLHniIkMPGgXKlJdvdP-NlJKNPo4wKOgfBA7t0CEYQCUaU2VQO_Chi8tMRrzDi2z_fvfuPcDrROJkXoHzkwg2zk2rJsZmWCz_RkoTzTCRmzbJ2xyTaRcSkGrVxqKvxzQpKk-81hczWYLWda363mfiGJ24ngRyrgvgqkhMqEl_FVdCDK-gjjJ6Kr8EdYmhJnSo2EJqlUdvrwBwtYBEIAmh2lHfGdCQyikmCmImPSTBm4mL2yZOF2j96fczIfWxpBkqzT4gVI2eIJVYsmRLzeo5OVdnyUGfWkPlDwW77Tb0eh6g08vb4wGrTX8G4Aq9xeawKJhwoRJtE8lo1MB61a5wiW8yO-aqsfUNOVvaJ-hotD_RSKFUruXu27p_6ZtKvsblqZKAd0PYmxFe23_Es3-nhCJc23Outen5PpXHQZdkYM_aoWKLsTiEGzzBAPUHYvr_4UfZ8GcJwp7A-ULE7mGDYNWxAgyAZSQ5oNauqyMW9TXpj17av6NcAM7wW8DcG6JIMaWQMNNxXPiFQEgBN0MuquYEXFbs9tFhA6wvMyrnAvX8YfyRUcp8rODP0DIxioP9GDmlvuCNgFQYMq6j8DNihTUy4SqZdJUFXyctdJQFXyQtc9dyFdeRpSUaxHWT7BN4G84a69eRu-RTa4vXtro5Bzw58L-iWMjmaroddcDZZ6lnDyQHDxHPHE_QAzScnI26bsxrBbLLEMMLFZD_pG1egb-o6MONoeOxzNxACRvYTyd3jmDyypZsX7FEOHTCW9NqfSytR1sxMe_mxKb8gfpGBViNITRswhHzb_OgZCtQE1t3kCgTnepDohpRH6APlkUPhIm7VIAOJX1WB01rgbXD4p9s4PkfLbY7YD3U2_1yEQH9SXwPn_VCfOR0SMhkSEgoJeXFICA4J-R9Ckox1Z4CBTqauWcl5vgbO41Nt3ZT37UQhu0e-cN6886T_bdJEs5YfqXqb54GLoFNYBGGFb5KCw2QkTc453fv4gVddgDyqxgs7DCmeMyikY_dLfvtEA9XcMXV9dx66nhsFIxhsXB7c09jTgNMnA5djbqP35_eo6XCTf_PktmYEOY0CzqGvXwfeAuCH4Ye_uiE38KkaEIWHBSwb5qKvrLDSZlxhWMhj6Zh00RDfCjaEAl0s43vleBEfWX2keRan8ZMK7SYWB3ZkmziVj92V-iZeANZftM6ptLBRe566dMTqsnuVf-8EyW_qxjtWzOcNf5ZqaCvK1Te-i9M9LRq2iNsqo4Ld5FTm7zhQK8r_KUu7ZlkuzyrvzU8A-peA538BA8wUgg)

**Comment by marlitas** [comment]
Hmmm I remember having an issue once with this with the onboarding doc. I don't remember the fix exactly but I'll investigate.

**Comment by marlitas** [comment]
Okay this is fixed. The github mermaid renderer is waaay more finicky than mermaid's live editor and apparently even IntelliJ's here's an issue I found that talks about error message improvement: https://github.com/mermaid-js/mermaid/issues/4388 ... although it might also be that github isn't fully up to date on the latest mermaid version so that syntax isn't something their API understands?

It was a very basic syntax change of moving a class definition on a node to a different spot. In order to debug I went line by line until the preview broke again. Not the most efficient but worked.

I also decreased the font size to work a bit better with the github preview window, but feel free to revert if that's not ideal.
